cypress/e2e/sample_test_spec/SampleTestRequest.cy.ts

```diff
@@ -0,0 +1,42 @@
+import { SampleTestPage } from "pageobject/Sample/SampleTestCreate";
+
+describe("Sample Test", () => {
+  const sampleTestPage = new SampleTestPage();
+  const sampleTestType = "BA/ETA",
+    icmrCategory = "Cat 0",
+    icmrLabel = "Test Icmr Label";
+
+  before(() => {
+    cy.loginByApi("devdistrictadmin", "Coronasafe@123");
+    cy.saveLocalStorage();
+  });
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.clearLocalStorage(/filters--.+/);
+  });
+
+  it("should request a new sample test", () => {
+    sampleTestPage.visitPatientPage();
+    sampleTestPage.visitPatientDashboardPage();
```

Comment:
```suggestion
    patientPage.visitPatient(patientName);
```

- reuse existing component, and use them in your test by importing the function
- always try to click on specific patient card, rather than random card
- keep all the input const in the test file itself